test/e2e/storage/volume_metrics.go

```diff
@@ -706,27 +702,9 @@ func verifyMetricCount(oldMetrics, newMetrics *storageControllerMetrics, metricN
 		oldLatencyCount = 0
 	}
 
-	oldStatusCount := int64(0)
```

Comment:
I think we still want to keep the counts check. The status count actually got moved to the existing "storage_operation_duration_seconds" metric.

Reply:
@gnufied: can you come up with a better fix?

I lack background about what these metrics do and I don't have time for something more complex than simply removing some apparently out-dated checks right now.

Reply:
@mauriciopoppe has gladly offered to take a look at this.